mail_tracking/static/src/xml/failed_message.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail_tracking.failed_message_items">
+        <div class="o_thread_date_separator o_border_dashed collapsed" data-toggle="collapse" data-target="#o_chatter_failed_message">
+            <a role="button" class="o_thread_date btn">
+                <i class="fa fa-fw fa-caret-down"/>
+                Failed messages
+                <small class="o_chatter_failed_message_summary ml8">
+                    <span class="badge rounded-circle badge-danger"><t t-esc="nbFailedMessages"/></span>
+                </small>
+            </a>
+        </div>
+        <div id="o_chatter_failed_message" class="in collapse">
+            <t t-foreach="failed_messages" t-as="message">
+                <div class="o_thread_message" style="margin-bottom: 10px">
+                    <div class="o_thread_message_sidebar">
+                        <div class="o_avatar_stack">
+                            <img t-attf-src="/web/image#{message.author[0] >= 0 ? ('/res.partner/' + message.author[0] + '/image_small') : ''}" class="o_thread_message_avatar rounded-circle mb8" t-att-title="message.author[1]" t-att-alt="message.author[1]"/>
```

Comment:
If I visit `/web/image/res.partner/-1/image_small` I get this image:

![image_small](https://user-images.githubusercontent.com/973709/67750875-f86be580-fa27-11e9-9068-6236d8965ddb.png)

It's basically the same as what you get from `/web/image`, so I think you can remove the logic here:

```suggestion
                            <img t-attf-src="/web/image#{message.author[0]}/image_small" class="o_thread_message_avatar rounded-circle mb8" t-att-title="message.author[1]" t-att-alt="message.author[1]"/>
```